bots/twitch/phantombot/egg-phantom-bot.json

```diff
@@ -1,24 +1,28 @@
 {
     "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
     "meta": {
-        "version": "PTDL_v1"
+        "version": "PTDL_v2",
+        "update_url": null
     },
-    "exported_at": "2020-12-16T19:18:24+02:00",
+    "exported_at": "2022-09-08T03:33:45+02:00",
     "name": "PhantomBot",
     "author": "mail@wuffy.eu",
     "description": "PhantomBot is an actively developed open source interactive Twitch bot with a vibrant community that provides entertainment and moderation for your channel, allowing you to focus on what matters the most to you - your game and your viewers.",
     "features": null,
-    "image": "quay.io\/parkervcp\/pterodactyl-images:debian_openjdk-11",
+    "docker_images": {
+        "quay.io\/parkervcp\/pterodactyl-images:debian_openjdk-11": "quay.io\/parkervcp\/pterodactyl-images:debian_openjdk-11"
```

Comment:
please use `ghcr.io/parkervcp/yolks:java_11` as the image your using is realy old and not updated